src/modules/dashboard/components/Dialogs/StartStreamDialog/StreamingDetails/constants.ts

```diff
@@ -0,0 +1,17 @@
+import { ExpenditureEndDateTypes } from '~pages/ExpenditurePage/types';
+
+export const user = {
+  id: '0xb77D57F4959eAfA0339424b83FcFaf9c15407461',
+  profile: {
+    avatarHash: null,
+    displayName: null,
+    username: 'WonderWoman',
+    walletAddress: '0xb77D57F4959eAfA0339424b83FcFaf9c15407461',
+  },
+};
+
+export const startDate = 1662988202074;
+
+export const endDateTime = 1663988202074;
+
+export const endDate = ExpenditureEndDateTypes.FixedTime;
```

Comment:
These shouldn't be needed if you are pulling in the data from the user entered fields.